types/restify/index.d.ts

```diff
@@ -946,6 +946,17 @@ export namespace plugins {
      */
     function authorizationParser(options?: any): RequestHandler;
 
+    interface HandlerCandidate {
+        handler: RequestHandler | RequestHandler[];
+        version?: string | string[];
+        contentType?: string | string[];
+    }
+
+    /**
+     * Runs first handler that matches to the condition
+     */
+    function conditionalHandler(candidates: HandlerCandidate | HandlerCandidate[]): RequestHandler[];
```

Comment:
The [docs](http://restify.com/docs/plugins-api/#conditionalhandler) you've linked indicate that the function returns a single function - typo here in the `.d.ts`? Or bad docs? Also a test would be nice, if you're not opposed.

Reply:
It appears the docs are wrong, it actually returns a candidate object. I looked mostly at the implementation to get this right, here: https://github.com/restify/node-restify/blob/master/lib/plugins/conditionalHandler.js#LC107

I'll issue restify to fix the doc.
And I'll look into adding a test.

Reply:
You're right, my bad. I'll fix that and see if I can a test